docs/api-reference/next/image.md

```diff
@@ -144,30 +116,32 @@ const MyImage = (props) => {
 
 ### sizes
 
-A string mapping media queries to device sizes. Defaults to `100vw`.
+A string that provides information about how wide the image will be at different breakpoints. Defaults to `100vw` (the full width of the screen).
+
+`sizes` is important for performance when using `layout="responsive"` or `layout="fill"` with images that take up less than the full viewport width.
```

Comment:
i think it could make sense to add an example for this to the demo app at [https://image-component.nextjs.gallery/](https://image-component.nextjs.gallery/).

Reply:
Lets remove this sentence, or merge it with the sentence below. If feels out of place as-is.

Reply:
@styfle sorry which sentence? The "defaults to 100vw" one?